forum/backend.py

```diff
@@ -11,6 +11,8 @@ def get_backend(
 ) -> Callable[[], MongoBackend | MySQLBackend]:
     """Return a factory function that lazily loads the backend API based on course_id."""
 
+    import pdb; pdb.set_trace();
```

Comment:
I know this is a draft but you'll probably want to remove this to get the tests to work :)